0.31.0: Limit user creation/ assignment to Teams (#1505)
astronomer/docs: software/import-idp-groups.md

After configuring your IdP group, at least one user belonging to the group has to log in to Software before the group appears as a Team in the Software UI. To quickly import many groups at once, add a user to all groups and then log in to Astronomer Software UI with the user's credentials.

After configuring and importing user groups, Workspace Admins and Deployment Admins can configure those groups as Teams using the Astronomer Software UI. To learn more about adding and setting permissions for Teams in the Astronomer Software UI, see [User Permissions](workspace-permissions.md#via-teams).

## Disable individual user management 

:::info

> This configuration requires setting up a root user. See [Create a root user].

Author: ```suggestion
This configuration requires setting up a root user. See [Create a root user].

:::
```

> To make it so that users can only join Astronomer Software and be assigned roles through Teams, set the following in your `config.yaml` file:

Author: ```suggestion
To make it so that users can join Astronomer Software and be assigned roles only through Teams, set the following in your `config.yaml` file:
```

After configuring your IdP group, at least one user belonging to the group has to log in to Software before the group appears as a Team in the Software UI. To quickly import many groups at once, add a user to all groups and then log in to Astronomer Software UI with the user's credentials.

After configuring and importing user groups, Workspace Admins and Deployment Admins can configure those groups as Teams using the Astronomer Software UI. To learn more about adding and setting permissions for Teams in the Astronomer Software UI, see [User Permissions](workspace-permissions.md#via-teams).

> ## Disable individual user management 

Reviewer: @cmarteepants @aliotta I am a little confused about this section. Can you please review it and ensure its essence and accuracy?

:::info

This configuration requires setting up a root user. See [Create a root user](create-a-root-user.md).

:::

> To provide user access to Astronomer Software and limit role assignment to Teams, add the following entry to your `config.yaml` file:

Author: ```suggestion
To use Teams as the only user management entity on Astronomer Software, add the following entry to your `config.yaml` file:
```

> To make it so that users can join Astronomer Software and be assigned roles only through Teams, add the following to your `config.yaml` file:

Reviewer 2: ```suggestion
To provide user access to Astronomer Software and limit user role assignment to Teams, add the following entry to your `config.yaml` file:
```

```yaml
astronomer: 
    houston:
     config:
       userManagement:

>          enabled: true

Author: ```suggestion
         enabled: false
```

```

> Save this configuration and push it to your platform as described in [Apply a Platform Config Change](apply-platform-config.md).

Reviewer 2: ```suggestion
Save this configuration and push it to your platform. See [Apply a Platform Config Change](apply-platform-config.md).
```

> After you apply the configuration, individual users can't be invited or assigned Workspace or Deployment-level roles in Astronomer Software. These users must be imported through a Team by a System Admin with access to your installation. Individual users can be assigned system-level roles only by your root user.

Author: ```suggestion
After you apply the configuration, individual users can't be invited or assigned Workspace or Deployment-level roles in Astronomer Software. Users must be invited through a Team by a System Admin, and only Teams can be assigned roles for Workspaces and Deployments. Individual users can be assigned system-level roles only by your root user.
```

> After you apply the configuration, individual users can't be invited or assigned Workspace or Deployment-level roles in Astronomer Software. These users must be imported through a Team by a System Admin to access your installation. Individual users can be assigned system-level roles only by your root user.

Reviewer 2: ```suggestion
After you apply the configuration, individual users can't be invited or assigned Workspace or Deployment-level roles in Astronomer Software. These users must be imported through a Team by a System Admin with access to your installation. Individual users can be assigned system-level roles only by your root user.
```
